{"name": "test_create_user_with_invalid_values", "status": "failed", "statusDetails": {"message": "AssertionError: Invalid input data\nassert 422 == 201", "trace": "set_environment_variable = 'stag'\n\n    @allure.severity(severity_level=\"Critical\")\n    def test_create_user_with_invalid_values(set_environment_variable):\n        endpoint = TestData().get_endpoint_env(\"stag\", 'create_user')\n        payload = TestData().payload_method()\n        header = TestData().header_method()\n    \n        payload['id'] = '5818268'\n        payload['name'] = ''\n        payload['email'] = 'davshakya@gmail.com'\n        payload['gender'] = 'tiger'\n        payload['status'] = ''\n    \n        payload_json = json.dumps(payload)\n        response = requests.request(\"POST\", endpoint, headers=header, data=payload_json)\n        response_code = response.status_code\n        res_dict = response.json()\n>       assert response_code == 201, \"Invalid input data\"\nE       AssertionError: Invalid input data\nE       assert 422 == 201\n\nTest_case\\test_create_user.py:57: AssertionError"}, "start": 1703345475155, "stop": 1703345475433, "uuid": "2bf03ddb-00ab-4022-a9d9-1b38129f520b", "historyId": "7245be0c793f44d7519ec76690baf3d8", "testCaseId": "7245be0c793f44d7519ec76690baf3d8", "fullName": "Test_case.test_create_user#test_create_user_with_invalid_values", "labels": [{"name": "severity", "value": "Critical"}, {"name": "parentSuite", "value": "Test_case"}, {"name": "suite", "value": "test_create_user"}, {"name": "host", "value": "devshakya"}, {"name": "thread", "value": "2660-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_case.test_create_user"}]}